dsa-solutions/lc-solutions/0000-0099/0030 - SubString-with-Concatenation-of-all-words.md

```diff
@@ -0,0 +1,377 @@
+---
+id: Substring with Concatenation of All Words
```

Comment:
replace `id: Substring with Concatenation of All Words` to `id: substring-with-concatenation-of-all-words`